.github/workflows/tekton_task_tests.yaml

```diff
@@ -19,6 +19,7 @@ jobs:
         with:
           files: |
             tasks/*/**
+            internal/tasks/*/**
```

Comment:
This was required as updating tasks in these two different "root dirs" it was not finding the ones in the "internal" directory".

Reply:
That's because `internal/tasks` is not the right spot for them. They should be in `tasks/internal`

Reply:
Got it. I will adjust the path.